WordPressCom-Stats-iOS/StatsEphemory.m

```diff
@@ -76,6 +76,19 @@ - (void)removeObjectForKey:(id)key
     [self.cache removeObjectForKey:key];
 }
 
+- (void)removeAllObjectsExpectObjectForKey:(id)key
```

Comment:
Please, forgive me if this is not correct. Shouldn't this method be named...

`removeAllObjectsExceptObjectWithKey` ? (Note: except instead of expect)